source/api_documentation/network_integration/advertiser_campaigns/index.rst

```diff
@@ -210,6 +210,22 @@ You are not allowed to delete campaigns.
     - integer
     - Maximum Promo Numbers.
 
+  * - **concurrent_call_cap_alert**
+    -
+    -
+
+  * - starts_at
+    - date
+    - Call Cap Start. Default: Beginning of day with the campaign's time zone.
+
```

Comment:
You are right, I have removed both `starts_at` and `time_zone` from the params list.